Crash when using 1.7.1

https://paste.ee/p/ls2eY
The same crash that occured on 1.7.0 is still plaguing the newer version
using forge 14.23.4.2705

Can confirm, but here is the crash report. Tested mods one by one and only crashed with the latest version of this mod.

---- Minecraft Crash Report ----

WARNING: coremods are present:
  ComfortsCoreLoadingPlugin (comforts-1.12.2-1.1.1.1.jar)
  AstralCore (astralsorcery-1.12.2-1.8.10.jar)
  ForgelinPlugin (Forgelin-1.7.1.jar)
  BCModPlugin (backpacks 1.12.2 - 3.5.2.jar)
  BedPatch (bedpatch-2.2-1.12.2.jar)
  CTMCorePlugin (CTM-MC1.12-0.3.0.15.jar)
  OpenModsCorePlugin (OpenModsLib-1.12.2-0.11.5.jar)
  LoadingPlugin (Quark-r1.4-123.jar)
  Do not report to Forge! Remove FoamFixAPI (or replace with FoamFixAPI-Lawful) and try again. (foamfix-0.9.9.1-1.12.2-anarchy.jar)
  SplashAnimationCoremod (SplashAnimation-0.1.0.jar)
  Plugin (NotEnoughIDs-1.5.4.2.jar)
  EnderCorePlugin (EnderCore-1.12.2-0.5.30.jar)
  CoreMod (Aroma1997Core-1.12.2-1.3.0.2.jar)
  MalisisCorePlugin (malisiscore-1.12.2-6.4.0.jar)
  ShetiPhian-ASM (shetiphiancore-1.12.0-3.5.6.jar)
  Inventory Tweaks Coremod (InventoryTweaks-1.63.jar)
  FMLPlugin (elulib-0.1.12.jar)
  IELoadingPlugin (ImmersiveEngineering-core-0.12-82.jar)
  SecretRoomsMod-Core (secretroomsmod-1.12.2-5.6.4.jar)
  LoadingPlugin (ResourceLoader-MC1.12.1-1.5.3.jar)
  TransformerLoader (OpenComputers-MC1.12.2-1.7.2.67.jar)
  IELoadingPlugin (ImmersiveEngineering-0.12-80-core.jar)
Contact their authors BEFORE contacting forge

// Quite honestly, I wouldn't worry myself about that.

Time: 6/10/18 9:32 AM
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderException: java.lang.ClassNotFoundException: com.feed_the_beast.ftbguides.FTBGuidesConfig
	at net.minecraftforge.fml.common.AutomaticEventSubscriber.inject(AutomaticEventSubscriber.java:89)
	at net.minecraftforge.fml.common.FMLModContainer.constructMod(FMLModContainer.java:602)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:218)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:196)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:135)
	at net.minecraftforge.fml.common.Loader.loadMods(Loader.java:593)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:225)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:466)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.ClassNotFoundException: com.feed_the_beast.ftbguides.FTBGuidesConfig
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at net.minecraftforge.fml.common.ModClassLoader.loadClass(ModClassLoader.java:75)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Unknown Source)
	at net.minecraftforge.fml.common.AutomaticEventSubscriber.inject(AutomaticEventSubscriber.java:81)
	... 36 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_171, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1894739744 bytes (1806 MB) / 2510290944 bytes (2394 MB) up to 11453595648 bytes (10923 MB)
	JVM Flags: 4 total; -Xmx12288m -Xms256m -XX:PermSize=256m -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.42 Powered by Forge 14.23.4.2707 Optifine OptiFine_1.12.2_HD_U_D3 246 mods loaded, 246 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State | ID                                | Version                  | Source                                            | Signature                                |
	|:----- |:--------------------------------- |:------------------------ |:------------------------------------------------- |:---------------------------------------- |
	| UC    | minecraft                         | 1.12.2                   | minecraft.jar                                     | None                                     |
	| UC    | mcp                               | 9.42                     | minecraft.jar                                     | None                                     |
	| UC    | FML                               | 8.0.99.99                | forge-1.12.2-14.23.4.2707.jar                     | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UC    | forge                             | 14.23.4.2707             | forge-1.12.2-14.23.4.2707.jar                     | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UC    | openmodscore                      | 0.11.5                   | minecraft.jar                                     | None                                     |
	| UC    | foamfixcore                       | 7.7.4                    | minecraft.jar                                     | None                                     |
	| UC    | opencomputers|core                | 1.7.2.67                 | minecraft.jar                                     | None                                     |
	| UC    | srm-hooks                         | 1.12.2-1.0.0             | minecraft.jar                                     | None                                     |
	| UC    | elucore                           | 1.0                      | minecraft.jar                                     | None                                     |
	| UC    | fastbench                         | 1.5.0                    | FastWorkbench-1.12.2-1.5.0.jar                    | None                                     |
	| UC    | actuallyadditions                 | 1.12.2-r135              | ActuallyAdditions-1.12.2-r135.jar                 | None                                     |
	| UC    | baubles                           | 1.5.2                    | Baubles-1.12-1.5.2.jar                            | None                                     |
	| UC    | actuallybaubles                   | 1.1                      | ActuallyBaubles-1.12-1.1.jar                      | None                                     |
	| UC    | redstoneflux                      | 2.0.2                    | RedstoneFlux-1.12-2.0.2.3-universal.jar           | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UC    | cofhcore                          | 4.5.2                    | CoFHCore-1.12.2-4.5.2.19-universal.jar            | None                                     |
	| UC    | libvulpes                         | 0.2.8.-32                | LibVulpes-1.12.2-0.2.8-32-universal.jar           | None                                     |
	| UC    | advancedrocketry                  | 1.4.0.-84                | AdvancedRocketry-1.12.2-1.4.0-84-universal.jar    | None                                     |
	| UC    | appliedenergistics2               | rv5-stable-11            | appliedenergistics2-rv5-stable-11.jar             | None                                     |
	| UC    | bdlib                             | 1.14.3.12                | bdlib-1.14.3.12-mc1.12.2.jar                      | None                                     |
	| UC    | ae2stuff                          | 0.7.0.4                  | ae2stuff-0.7.0.4-mc1.12.2.jar                     | None                                     |
	| UC    | akashictome                       | 1.2-10                   | AkashicTome-1.2-10.jar                            | None                                     |
	| UC    | crafttweaker                      | 4.1.8                    | CraftTweaker2-1.12-4.1.8.jar                      | None                                     |
	| UC    | mtlib                             | 3.0.5                    | MTLib-3.0.5.jar                                   | None                                     |
	| UC    | modtweaker                        | 4.0.12                   | modtweaker-4.0.12.jar                             | None                                     |
	| UC    | jei                               | 4.9.2.196                | jei_1.12.2-4.9.2.196.jar                          | None                                     |
	| UC    | appleskin                         | 1.0.9                    | AppleSkin-mc1.12-1.0.9.jar                        | None                                     |
	| UC    | architecturecraft                 | @VERSION@                | architecturecraft-1.12-2.68.jar                   | None                                     |
	| UC    | aroma1997core                     | 1.3.0.2                  | Aroma1997Core-1.12.2-1.3.0.2.jar                  | dfbfe4c473253d8c5652417689848f650b2cbe32 |
	| UC    | aroma1997sdimension               | 1.3.0.0.b37              | Aroma1997s-Dimensional-World-1.12-1.3.0.0.b37.jar | None                                     |
	| UC    | astralsorcery                     | 1.8.10                   | astralsorcery-1.12.2-1.8.10.jar                   | a0f0b759d895c15ceb3e3bcb5f3c2db7c582edf0 |
	| UC    | morphtool                         | 1.2-16                   | Morph-o-Tool-1.2-16.jar                           | None                                     |
	| UC    | quark                             | r1.4-123                 | Quark-r1.4-123.jar                                | None                                     |
	| UC    | autoreglib                        | 1.3-17                   | AutoRegLib-1.3-17.jar                             | None                                     |
	| UC    | backpacks16840                    | 3.5.2                    | backpacks 1.12.2 - 3.5.2.jar                      | None                                     |
	| UC    | badwithernocookiereloaded         | 1.12.2-2.12              | badwithernocookiereloaded-1.12.2-2.12.jar         | None                                     |
	| UC    | base                              | 3.8.1                    | base-1.12.2-3.8.1.jar                             | None                                     |
	| UC    | betteradvancements                | 0.0.5.17                 | BetterAdvancements-1.12.2-0.0.5.17.jar            | None                                     |
	| UC    | betterbuilderswands               | 0.11.1                   | BetterBuildersWands-1.12-0.11.1.245+69d0d70.jar   | None                                     |
	| UC    | bibliocraft                       | 2.4.4                    | BiblioCraft[v2.4.4][MC1.12.2].jar                 | None                                     |
	| UC    | ic2                               | 2.8.73-ex112             | industrialcraft-2-2.8.73-ex112.jar                | de041f9f6187debbc77034a344134053277aa3b0 |
	| UC    | mantle                            | 1.12-1.3.2.24            | Mantle-1.12-1.3.2.24.jar                          | None                                     |
	| UC    | natura                            | 1.12.2-4.3.2.49          | natura-1.12.2-4.3.2.49.jar                        | None                                     |
	| UC    | forestry                          | 5.8.0.295                | forestry_1.12.2-5.8.0.295.jar                     | None                                     |
	| UC    | binniecore                        | 2.5.0.165                | binnie-mods-1.12.2-2.5.0.165.jar                  | None                                     |
	| UC    | binniedesign                      | 2.5.0.165                | binnie-mods-1.12.2-2.5.0.165.jar                  | None                                     |
	| UC    | genetics                          | 2.5.0.165                | binnie-mods-1.12.2-2.5.0.165.jar                  | None                                     |
	| UC    | botany                            | 2.5.0.165                | binnie-mods-1.12.2-2.5.0.165.jar                  | None                                     |
	| UC    | extrabees                         | 2.5.0.165                | binnie-mods-1.12.2-2.5.0.165.jar                  | None                                     |
	| UC    | extratrees                        | 2.5.0.165                | binnie-mods-1.12.2-2.5.0.165.jar                  | None                                     |
	| UC    | blockcraftery                     | 0.1.3                    | blockcraftery-0.1.3.jar                           | None                                     |
	| UC    | guideapi                          | 1.12-2.1.5-60            | Guide-API-1.12-2.1.5-60.jar                       | None                                     |
	| UC    | bloodmagic                        | 1.12.2-2.2.12-97         | BloodMagic-1.12.2-2.2.12-97.jar                   | None                                     |
	| UC    | bookshelf                         | 2.3.544                  | Bookshelf-1.12.2-2.3.544.jar                      | d476d1b22b218a10d845928d1665d45fce301b27 |
	| UC    | thaumcraft                        | 6.1.BETA16               | Thaumcraft-1.12.2-6.1.BETA16.jar                  | None                                     |
	| UC    | botania                           | r1.10-354                | Botania r1.10-354.jar                             | None                                     |
	| UC    | codechickenlib                    | 3.1.9.344                | CodeChickenLib-1.12.2-3.1.9.344-universal.jar     | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| UC    | brandonscore                      | 2.4.2                    | BrandonsCore-1.12-2.4.2.157-universal.jar         | None                                     |
	| UC    | careerbees                        | 1.0                      | careerbees-0.4.0.jar                              | None                                     |
	| UC    | chameleon                         | 1.12-4.1.3               | Chameleon-1.12-4.1.3.jar                          | None                                     |
	| UC    | chesttransporter                  | 2.8.8                    | ChestTransporter-1.12.2-2.8.8.jar                 | None                                     |
	| UC    | chickenchunks                     | 2.4.1.71                 | ChickenChunks-1.12.2-2.4.1.71-universal.jar       | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| UC    | ctm                               | MC1.12-0.3.0.15          | CTM-MC1.12-0.3.0.15.jar                           | None                                     |
	| UC    | chisel                            | MC1.12.2-0.2.0.31        | Chisel-MC1.12.2-0.2.0.31.jar                      | None                                     |
	| UC    | chiselsandbits                    | 14.17                    | chiselsandbits-14.17.jar                          | None                                     |
	| UC    | clumps                            | 2.0.0                    | Clumps-3.0.0.jar                                  | None                                     |
	| UC    | cofhworld                         | 1.2.0                    | CoFHWorld-1.12.2-1.2.0.5-universal.jar            | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UC    | morpheus                          | 1.12-3.3.2               | Morpheus-1.12-3.3.2.jar                           | None                                     |
	| UC    | comforts                          | 1.1.1.1                  | comforts-1.12.2-1.1.1.1.jar                       | 5d5b8aee896a4f5ea3f3114784742662a67ad32f |
	| UC    | cyclopscore                       | 0.11.6                   | CyclopsCore-1.12.2-0.11.6.jar                     | bd0353b3e8a2810d60dd584e256e364bc3bedd44 |
	| UC    | commoncapabilities                | 1.4.0                    | CommonCapabilities-1.12-1.4.0.jar                 | None                                     |
	| UC    | compacter                         | 1.3.0.3                  | compacter-1.3.0.3-mc1.12.2.jar                    | None                                     |
	| UC    | compactmachines3                  | 3.0.12                   | compactmachines3-1.12.2-3.0.12-b215.jar           | None                                     |
	| UC    | compactsolars                     | 1.12.2-5.0.17.340        | CompactSolars-1.12.2-5.0.17.340-universal.jar     | None                                     |
	| UC    | endercore                         | 1.12.2-0.5.30            | EnderCore-1.12.2-0.5.30.jar                       | None                                     |
	| UC    | thermalfoundation                 | 2.5.0                    | ThermalFoundation-1.12.2-2.5.0.19-universal.jar   | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UC    | draconicevolution                 | 2.3.11                   | Draconic-Evolution-1.12-2.3.11.290-universal.jar  | None                                     |
	| UC    | thermalexpansion                  | 5.5.0                    | ThermalExpansion-1.12.2-5.5.0.29-universal.jar    | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UC    | enderio                           | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | enderiointegrationtic             | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | tconstruct                        | 1.12.2-2.10.1.84         | TConstruct-1.12.2-2.10.1.84.jar                   | None                                     |
	| UC    | conarm                            | 0.0.21-rc1               | conarm-1.12.2-0.0.21-rc1.jar                      | 5d5b8aee896a4f5ea3f3114784742662a67ad32f |
	| UC    | controlling                       | 3.0.6                    | Controlling-3.0.6.jar                             | None                                     |
	| UC    | cookingforblockheads              | 6.4.39                   | CookingForBlockheads_1.12.2-6.4.39.jar            | None                                     |
	| UC    | ctgui                             | 1.0.0                    | CraftTweaker2-1.12-4.1.8.jar                      | None                                     |
	| UC    | crafttweakerjei                   | 2.0.2                    | CraftTweaker2-1.12-4.1.8.jar                      | None                                     |
	| UC    | cucumber                          | 1.1.0                    | cucumber-1.12-1.1.0.jar                           | None                                     |
	| UC    | custommainmenu                    | 2.0.8                    | CustomMainMenu-MC1.12.2-2.0.8.jar                 | None                                     |
	| UC    | p455w0rdslib                      | 2.0.29                   | p455w0rdslib-1.12-2.0.29.jar                      | None                                     |
	| UC    | theoneprobe                       | 1.4.23                   | theoneprobe-1.12-1.4.23.jar                       | None                                     |
	| UC    | danknull                          | 1.4.33                   | DankNull-1.12-1.4.33.jar                          | None                                     |
	| UC    | darkutils                         | 1.8.207                  | DarkUtils-1.12.2-1.8.207.jar                      | d476d1b22b218a10d845928d1665d45fce301b27 |
	| UC    | eleccore                          | 1.8.434                  | ElecCore-1.12.2-1.8.434.jar                       | None                                     |
	| UC    | mcjtylib_ng                       | 3.0.0                    | mcjtylib-1.12-3.0.0.jar                           | None                                     |
	| UC    | opencomputers                     | 1.7.2.67                 | OpenComputers-MC1.12.2-1.7.2.67.jar               | None                                     |
	| UC    | deepresonance                     | 1.7.0                    | deepresonance-1.12-1.7.0.jar                      | None                                     |
	| UC    | journeymap                        | 1.12.2-5.5.2             | journeymap-1.12.2-5.5.2.jar                       | None                                     |
	| UC    | defaultoptions                    | 9.2.7                    | DefaultOptions_1.12.2-9.2.7.jar                   | None                                     |
	| UC    | diethopper                        | 1.1                      | diethopper-1.1.jar                                | None                                     |
	| UC    | dirt2path                         | 1.7.1                    | dirt2path-1.7.1.jar                               | None                                     |
	| UC    | elevatorid                        | 1.3.6                    | ElevatorMod-1.12.2-1.3.6.jar                      | None                                     |
	| UC    | embers                            | 0.230                    | embers-0.230.jar                                  | None                                     |
	| UC    | enderiobase                       | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | enderioconduits                   | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | enderioconduitsappliedenergistics | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | enderioconduitsopencomputers      | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | enderioconduitsrefinedstorage     | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | enderiointegrationforestry        | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | enderiointegrationticlate         | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | mcmultipart                       | 2.5.1                    | MCMultiPart-2.5.1.jar                             | None                                     |
	| UC    | ftblib                            | 5.1.0.22                 | FTBLib-5.1.0.22.jar                               | None                                     |
	| UC    | enderiomachines                   | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | enderiopowertools                 | 5.0.26                   | EnderIO-1.12.2-5.0.26.jar                         | None                                     |
	| UC    | enderrift                         | 2.1.9                    | EnderRift-1.12.1-2.1.9.jar                        | None                                     |
	| UC    | enderstorage                      | 2.4.3.131                | EnderStorage-1.12.2-2.4.3.131-universal.jar       | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| UC    | immersiveengineering              | 0.12-82                  | ImmersiveEngineering-0.12-82.jar                  | 4cb49fcde3b43048c9889e0a3d083225da926334 |
	| UC    | engineersdoors                    | 0.7.2                    | engineersdoors-0.7.2.jar                          | None                                     |
	| UC    | reborncore                        | 3.8.8.298                | RebornCore-1.12.2-3.8.8.298-universal.jar         | 8727a3141c8ec7f173b87aa78b9b9807867c4e6b |
	| UC    | engineersworkshop                 | 1.0.8.37                 | EngineersWorkshopReborn-1.12.2-1.0.8.37.jar       | 8727a3141c8ec7f173b87aa78b9b9807867c4e6b |
	| UC    | valkyrielib                       | 1.12.2-2.0.11a           | valkyrielib-1.12.2-2.0.11a.jar                    | None                                     |
	| UC    | environmentaltech                 | 1.12.2-2.0.11a           | environmentaltech-1.12.2-2.0.11a.jar              | None                                     |
	| UC    | etlunar                           | 1.12.2-2.0.11a           | etlunar-1.12.2-2.0.11a.jar                        | None                                     |
	| UC    | exchangers                        | 1.12.2-2.7.2             | Exchangers-1.12.2-2.7.2.jar                       | 4ffa87db52cf086d00ecc4853a929367b1c39b5c |
	| UC    | extracells                        | 2.5.13                   | ExtraCells-1.12.2-2.5.13a58.jar                   | None                                     |
	| UC    | shadowmc                          | 3.8.0                    | ShadowMC-1.12-3.8.0.jar                           | None                                     |
	| UC    | extrarails                        | 1.3.0                    | ExtraRails-1.12-1.3.0.jar                         | None                                     |
	| UC    | extrautils2                       | 1.0                      | extrautils2-1.12-1.7.6.jar                        | None                                     |
	| UC    | zerocore                          | 1.12-0.1.2.2             | zerocore-1.12-0.1.2.2.jar                         | None                                     |
	| UC    | bigreactors                       | 1.12.2-0.4.5.49          | ExtremeReactors-1.12.2-0.4.5.49.jar               | None                                     |
	| UC    | fencejumper                       | 1.0.3                    | fencejumper-1.12-1.0.3.jar                        | None                                     |
	| UC    | flatcoloredblocks                 | mc1.12-6.6               | flatcoloredblocks-mc1.12-6.6.jar                  | None                                     |
	| UC    | sonarcore                         | 5.0.12                   | sonarcore-1.12.2-5.0.12.jar                       | None                                     |
	| UC    | fluxnetworks                      | 3.0.13                   | fluxnetworks-1.12.2-3.0.13.jar                    | None                                     |
	| UC    | foamfix                           | 0.9.9.1-1.12.2           | foamfix-0.9.9.1-1.12.2-anarchy.jar                | None                                     |
	| UC    | forgelin                          | 1.7.1                    | Forgelin-1.7.1.jar                                | None                                     |
	| UC    | forgemultipartcbe                 | 2.4.2.58                 | ForgeMultipart-1.12.2-2.4.2.58-universal.jar      | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| UC    | microblockcbe                     | 2.4.2.58                 | ForgeMultipart-1.12.2-2.4.2.58-universal.jar      | None                                     |
	| UC    | minecraftmultipartcbe             | 2.4.2.58                 | ForgeMultipart-1.12.2-2.4.2.58-universal.jar      | None                                     |
	| UE    | ftbguides                         | 1.0.2.10                 | FTBGuides-1.0.2.10.jar                            | None                                     |
	| U     | ftbutilities                      | 5.1.0.27                 | FTBUtilities-5.1.0.27.jar                         | None                                     |
	| U     | gadgetrycore                      | 0.2.0                    | gadgetrycore-0.2.0.jar                            | None                                     |
	| U     | gadgetrymachines                  | 0.2.1                    | gadgetrymachines-0.2.1.jar                        | None                                     |
	| U     | gendustry                         | 1.6.5.8                  | gendustry-1.6.5.8-mc1.12.2.jar                    | None                                     |
	| U     | gravestone                        | 1.10.1                   | gravestone-1.10.1.jar                             | None                                     |
	| U     | harvest                           | 1.12-1.2.6-18            | Harvest-1.12-1.2.6-18.jar                         | None                                     |
	| U     | immersivepetroleum                | 1.1.9                    | immersivepetroleum-1.12.2-1.1.9.jar               | None                                     |
	| U     | teslacorelib                      | 1.0.14                   | tesla-core-lib-1.12.2-1.0.14.jar                  | None                                     |
	| U     | industrialforegoing               | 1.12.2-1.12.2            | industrialforegoing-1.12.2-1.10.0-173.jar         | None                                     |
	| U     | industrialwires                   | 1.6-25                   | IndustrialWires-1.6-25.jar                        | None                                     |
	| U     | integrateddynamics                | 0.11.12                  | IntegratedDynamics-1.12.2-0.11.12.jar             | None                                     |
	| U     | integrateddynamicscompat          | 1.0.0                    | IntegratedDynamics-1.12.2-0.11.12.jar             | None                                     |
	| U     | integratedtunnels                 | 1.5.6                    | IntegratedTunnels-1.12.2-1.5.6.jar                | None                                     |
	| U     | integratedtunnelscompat           | 1.0.0                    | IntegratedTunnels-1.12.2-1.5.6.jar                | None                                     |
	| U     | inventorytweaks                   | 1.63+release.109.220f184 | InventoryTweaks-1.63.jar                          | None                                     |
	| U     | ironbackpacks                     | 1.12.2-3.0.8-12          | IronBackpacks-1.12.2-3.0.8-12.jar                 | None                                     |
	| U     | ironchest                         | 1.12.2-7.0.40.824        | ironchest-1.12.2-7.0.40.824.jar                   | None                                     |
	| U     | harvestcraft                      | 1.12.2u                  | Pam's HarvestCraft 1.12.2u.jar                    | None                                     |
	| U     | jehc                              | 1.12.2                   | JEHC-1.12.2-1.5.1.0.jar                           | None                                     |
	| U     | jeibees                           | 0.9.0.5                  | jeibees-0.9.0.5-mc1.12.2.jar                      | None                                     |
	| U     | jeiintegration                    | 1.5.1.36                 | jeiintegration_1.12.2-1.5.1.36.jar                | None                                     |
	| U     | jeresources                       | 0.8.5.27                 | JustEnoughResources-1.12.2-0.8.5.27.jar           | None                                     |
	| U     | kleeslabs                         | 5.4.10                   | KleeSlabs_1.12.2-5.4.10.jar                       | None                                     |
	| U     | llor                              | 1.1.6-mc1.12.2           | LLOverlayReloaded-1.1.6-mc1.12.2.jar              | None                                     |
	| U     | longfallboots                     | 1.2.1a                   | longfallboots-1.2.1a.jar                          | None                                     |
	| U     | lostcities                        | 2.0.9                    | lostcities-1.12-2.0.9.jar                         | None                                     |
	| U     | magicbees                         | 1.0                      | MagicBees-1.12.2-3.1.10.jar                       | None                                     |
	| U     | malisiscore                       | 1.12.2-6.4.0             | malisiscore-1.12.2-6.4.0.jar                      | None                                     |
	| U     | malisisdoors                      | 1.12.2-7.3.0             | malisisdoors-1.12.2-7.3.0.jar                     | None                                     |
	| U     | minetogether                      | unspecified              | minetogether-1.10.2-1.8.7.jar                     | None                                     |
	| U     | minetogetherserver                | unspecified              | minetogether-1.10.2-1.8.7.jar                     | None                                     |
	| U     | mob_grinding_utils                | 0.3.6                    | MobGrindingUtils-0.3.6.jar                        | None                                     |
	| U     | modnametooltip                    | 1.10.0                   | modnametooltip_1.12.2-1.10.0.jar                  | None                                     |
	| U     | moreoverlays                      | 1.13                     | moreoverlays-1.13-mc1.12.1.jar                    | None                                     |
	| U     | mousetweaks                       | 2.8                      | MouseTweaks-2.8-mc1.12.1.jar                      | None                                     |
	| U     | mrtjpcore                         | 2.1.2.34                 | MrTJPCore-1.12.2-2.1.2.34-universal.jar           | None                                     |
	| U     | mysticalagriculture               | 1.6.10                   | mysticalagriculture-1.12-1.6.10.jar               | None                                     |
	| U     | mysticalagradditions              | 1.2.8                    | mysticalagradditions-1.12-1.2.8.jar               | None                                     |
	| U     | naturescompass                    | 1.5.1                    | NaturesCompass-1.12.2-1.5.1.jar                   | None                                     |
	| U     | netherportalfix                   | 5.3.13                   | NetherPortalFix_1.12.1-5.3.13.jar                 | None                                     |
	| U     | nice                              | 0.3.0                    | nice-1.12-0.3.0.jar                               | None                                     |
	| U     | neid                              | 1.5.4.2                  | NotEnoughIDs-1.5.4.2.jar                          | None                                     |
	| U     | notenoughwands                    | 1.7.0                    | notenoughwands-1.12-1.7.0.jar                     | None                                     |
	| U     | nuclearcraft                      | 2.10p                    | NuclearCraft-2.10p--1.12.2.jar                    | None                                     |
	| U     | openmods                          | 0.11.5                   | OpenModsLib-1.12.2-0.11.5.jar                     | None                                     |
	| U     | openblocks                        | 1.7.6                    | OpenBlocks-1.12.2-1.7.6.jar                       | None                                     |
	| U     | placebo                           | 1.3.4                    | Placebo-1.12.2-1.3.4.jar                          | None                                     |
	| U     | shetiphiancore                    | 3.5.6                    | shetiphiancore-1.12.0-3.5.6.jar                   | None                                     |
	| U     | platforms                         | 1.4.5                    | platforms-1.12.0-1.4.5.jar                        | None                                     |
	| U     | projectred-core                   | 4.9.0.86                 | ProjectRed-1.12.2-4.9.0.86-Base.jar               | None                                     |
	| U     | projectred-integration            | 4.9.0.86                 | ProjectRed-1.12.2-4.9.0.86-integration.jar        | None                                     |
	| U     | projectred-transmission           | 4.9.0.86                 | ProjectRed-1.12.2-4.9.0.86-integration.jar        | None                                     |
	| U     | projectred-fabrication            | 4.9.0.86                 | ProjectRed-1.12.2-4.9.0.86-fabrication.jar        | None                                     |
	| U     | projectred-illumination           | 4.9.0.86                 | ProjectRed-1.12.2-4.9.0.86-lighting.jar           | None                                     |
	| U     | projectred-expansion              | 1.0                      | ProjectRed-1.12.2-4.9.0.86-mechanical.jar         | None                                     |
	| U     | projectred-relocation             | 1.0                      | ProjectRed-1.12.2-4.9.0.86-mechanical.jar         | None                                     |
	| U     | projectred-transportation         | 1.0                      | ProjectRed-1.12.2-4.9.0.86-mechanical.jar         | None                                     |
	| U     | projectred-exploration            | 4.9.0.86                 | ProjectRed-1.12.2-4.9.0.86-world.jar              | None                                     |
	| U     | quickleafdecay                    | 1.2.4                    | QuickLeafDecay-MC1.12.1-1.2.4.jar                 | None                                     |
	| U     | rangedpumps                       | 0.5                      | rangedpumps-0.5.jar                               | None                                     |
	| U     | redstonearsenal                   | 2.5.1                    | RedstoneArsenal-1.12.2-2.5.1.13-universal.jar     | None                                     |
	| U     | xreliquary                        | 1.12.2-1.3.4.763         | Reliquary-1.12.2-1.3.4.763.jar                    | None                                     |
	| U     | resourceloader                    | 1.5.3                    | ResourceLoader-MC1.12.1-1.5.3.jar                 | None                                     |
	| U     | rftools                           | 7.50                     | rftools-1.12-7.51.jar                             | None                                     |
	| U     | rftoolscontrol                    | 1.9.0                    | rftoolsctrl-1.12-1.9.0.jar                        | None                                     |
	| U     | rftoolspower                      | 1.1.0                    | rftoolspower-1.12-1.1.0.jar                       | None                                     |
	| U     | roots                             | 0.104                    | roots-2-0.104.jar                                 | None                                     |
	| U     | rustic                            | 1.0.5                    | rustic-1.0.5.jar                                  | None                                     |
	| U     | secretroomsmod                    | 5.6.4                    | secretroomsmod-1.12.2-5.6.4.jar                   | None                                     |
	| U     | signals                           | 1.0                      | Signals-1.12.2-1.2.2-5-universal.jar              | None                                     |
	| U     | simplevoidworld                   | 1.2.0.9                  | Simple-Void-World-1.12-1.2.0.9-universal.jar      | None                                     |
	| U     | thermaldynamics                   | 2.5.1                    | ThermalDynamics-1.12.2-2.5.1.14-universal.jar     | None                                     |
	| U     | simplyjetpacks                    | 2.2.5.39                 | SimplyJetpacks2-1.12.2-2.2.5.39.jar               | None                                     |
	| U     | snad                              | 1.12.1-1.7.09.16a        | Snad-1.12.1-1.7.09.16a.jar                        | None                                     |
	| U     | soulshardstow                     | 1.12-2.7.6-56            | SoulShards-TOW-1.12-2.7.6-56.jar                  | None                                     |
	| U     | stevescarts                       | 2.4.20.99                | StevesCarts-1.12.2-2.4.20.99.jar                  | None                                     |
	| U     | storagedrawers                    | 1.12-5.3.5               | StorageDrawers-1.12.2-5.3.6.jar                   | None                                     |
	| U     | storagedrawersextra               | @VERSION@                | StorageDrawersExtras-1.12-3.1.0.jar               | None                                     |
	| U     | tcinventoryscan                   | 2.0.10                   | ThaumicInventoryScanning_1.12.2-2.0.10.jar        | None                                     |
	| U     | thaumicjei                        | 1.2.1                    | ThaumicJEI-1.12.2-1.3.1-12.jar                    | None                                     |
	| U     | thermalcultivation                | 0.3.0                    | ThermalCultivation-1.12.2-0.3.0.7-universal.jar   | None                                     |
	| U     | thermalinnovation                 | 0.3.0                    | ThermalInnovation-1.12.2-0.3.0.7-universal.jar    | None                                     |
	| U     | thutcore                          | 5.16.3                   | thutcore-1.12.2-5.16.3.jar                        | None                                     |
	| U     | thutcore_compat                   | 1.0                      | thutcore-1.12.2-5.16.3.jar                        | None                                     |
	| U     | thuttech                          | 6.0.16                   | thuttech-1.12.2-6.0.16.jar                        | None                                     |
	| U     | tinkersaddons                     | 1.0.7                    | Tinkers'+Addons-1.12.1-1.0.7.jar                  | None                                     |
	| U     | tinkertoolleveling                | 1.12-1.0.4.DEV.5d716b9   | TinkerToolLeveling-1.12-1.0.4.jar                 | None                                     |
	| U     | tipthescales                      | 1.0.1                    | TipTheScales-1.12.2-1.0.1.jar                     | None                                     |
	| U     | topaddons                         | 1.12.2-1.7.0             | topaddons-1.12.2-1.7.0.jar                        | None                                     |
	| U     | torchmaster                       | 1.6.4.62                 | torchmaster_1.12.2-1.6.4.62.jar                   | None                                     |
	| U     | translocators                     | 2.5.0.70                 | Translocators-1.12.2-2.5.0.70-universal.jar       | None                                     |
	| U     | traverse                          | 1.5.3                    | Traverse-1.12.2-1.5.3-58.jar                      | None                                     |
	| U     | twilightforest                    | 3.7.424                  | twilightforest-1.12.2-3.7.424-universal.jar       | None                                     |
	| U     | universalmodifiers                | 1.12.2-1.0.9a            | valkyrielib-1.12.2-2.0.11a.jar                    | None                                     |
	| U     | wanionlib                         | 1.12.2-1.5               | WanionLib-1.12.2-1.5.jar                          | None                                     |
	| U     | waystones                         | 4.0.43                   | Waystones_1.12.2-4.0.43.jar                       | None                                     |
	| U     | whoosh                            | 0.1.10                   | Whoosh-1.12.2-0.1.11.1-universal.jar              | None                                     |
	| U     | wct                               | 3.9.64                   | WirelessCraftingTerminal-1.12.2-3.9.64.jar        | None                                     |
	| U     | woot                              | 1.12.2-1.3.0             | woot-1.12.2-1.3.0.jar                             | None                                     |
	| U     | wrcbe                             | 2.3.0                    | WR-CBE-1.12.2-2.3.0.27-universal.jar              | None                                     |
	| U     | xnet                              | 1.7.0                    | xnet-1.12-1.7.0.jar                               | None                                     |
	| U     | xray                              | 1.4.0                    | xray-1.12.2-1.4.2.jar                             | None                                     |
	| U     | xtones                            | 1.12-1.0.8-11            | Xtones-1.12-1.0.8-11.jar                          | None                                     |
	| U     | yabba                             | 1.1.2.8                  | YABBA-1.1.2.8.jar                                 | None                                     |
	| U     | reauth                            | 3.6.0                    | reauth-3.6.0.jar                                  | None                                     |
	| U     | elulib                            | 0.1.12                   | elulib-0.1.12.jar                                 | None                                     |
	| U     | teslacorelib_registries           | 1.0.14                   | tesla-core-lib-1.12.2-1.0.14.jar                  | None                                     |
	| U     | unidict                           | 1.12.2-2.5c              | UniDict-1.12.2-2.5c.jar                           | None                                     |

	Loaded coremods (and transformers): 
ComfortsCoreLoadingPlugin (comforts-1.12.2-1.1.1.1.jar)
  c4.comforts.asm.SleepTransformer
AstralCore (astralsorcery-1.12.2-1.8.10.jar)
  
ForgelinPlugin (Forgelin-1.7.1.jar)
  
BCModPlugin (backpacks 1.12.2 - 3.5.2.jar)
  brad16840.core.ClassTransformer
BedPatch (bedpatch-2.2-1.12.2.jar)
  com.mordenkainen.bedpatch.BedPatchASM
CTMCorePlugin (CTM-MC1.12-0.3.0.15.jar)
  team.chisel.ctm.client.asm.CTMTransformer
OpenModsCorePlugin (OpenModsLib-1.12.2-0.11.5.jar)
  openmods.core.OpenModsClassTransformer
LoadingPlugin (Quark-r1.4-123.jar)
  vazkii.quark.base.asm.ClassTransformer
Do not report to Forge! Remove FoamFixAPI (or replace with FoamFixAPI-Lawful) and try again. (foamfix-0.9.9.1-1.12.2-anarchy.jar)
  pl.asie.foamfix.coremod.FoamFixTransformer
SplashAnimationCoremod (SplashAnimation-0.1.0.jar)
  pl.asie.splashanimation.core.SplashProgressTransformer
Plugin (NotEnoughIDs-1.5.4.2.jar)
  ru.fewizz.neid.asm.Transformer
EnderCorePlugin (EnderCore-1.12.2-0.5.30.jar)
  com.enderio.core.common.transform.EnderCoreTransformer
  com.enderio.core.common.transform.SimpleMixinPatcher
CoreMod (Aroma1997Core-1.12.2-1.3.0.2.jar)
  
MalisisCorePlugin (malisiscore-1.12.2-6.4.0.jar)
  
ShetiPhian-ASM (shetiphiancore-1.12.0-3.5.6.jar)
  shetiphian.asm.ClassTransformer
Inventory Tweaks Coremod (InventoryTweaks-1.63.jar)
  invtweaks.forge.asm.ContainerTransformer
FMLPlugin (elulib-0.1.12.jar)
  elucent.elulib.asm.ASMTransformer
IELoadingPlugin (ImmersiveEngineering-core-0.12-82.jar)
  blusunrize.immersiveengineering.common.asm.IEClassTransformer
SecretRoomsMod-Core (secretroomsmod-1.12.2-5.6.4.jar)
  com.wynprice.secretroomsmod.core.SecretRoomsTransformer
LoadingPlugin (ResourceLoader-MC1.12.1-1.5.3.jar)
  lumien.resourceloader.asm.ClassTransformer
TransformerLoader (OpenComputers-MC1.12.2-1.7.2.67.jar)
  li.cil.oc.common.asm.ClassTransformer
IELoadingPlugin (ImmersiveEngineering-0.12-80-core.jar)
  blusunrize.immersiveengineering.common.asm.IEClassTransformer
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 398.11' Renderer: 'GeForce GTX 750 Ti/PCIe/SSE2'
	OpenModsLib class transformers: [llama_null_fix:FINISHED],[horse_base_null_fix:FINISHED],[pre_world_render_hook:FINISHED],[player_render_hook:FINISHED],[horse_null_fix:FINISHED]
	AE2 Version: stable rv5-stable-11 for Forge 14.23.1.2554
	Pulsar/natura loaded Pulses: 
		- NaturaCommons (Enabled/Forced)
		- NaturaOverworld (Enabled/Not Forced)
		- NaturaNether (Enabled/Not Forced)
		- NaturaDecorative (Enabled/Not Forced)
		- NaturaTools (Enabled/Not Forced)
		- NaturaEntities (Enabled/Not Forced)
		- NaturaOredict (Enabled/Forced)
		- NaturaWorld (Enabled/Not Forced)

	Pulsar/tconstruct loaded Pulses: 
		- TinkerCommons (Enabled/Forced)
		- TinkerWorld (Enabled/Not Forced)
		- TinkerTools (Enabled/Not Forced)
		- TinkerHarvestTools (Enabled/Forced)
		- TinkerMeleeWeapons (Enabled/Forced)
		- TinkerRangedWeapons (Enabled/Forced)
		- TinkerModifiers (Enabled/Forced)
		- TinkerSmeltery (Enabled/Not Forced)
		- TinkerGadgets (Enabled/Not Forced)
		- TinkerOredict (Enabled/Forced)
		- TinkerIntegration (Enabled/Forced)
		- TinkerFluids (Enabled/Forced)
		- TinkerMaterials (Enabled/Forced)
		- TinkerModelRegister (Enabled/Forced)
		- chiselIntegration (Enabled/Not Forced)
		- chiselsandbitsIntegration (Enabled/Not Forced)
		- theoneprobeIntegration (Enabled/Not Forced)

	OptiFine Version: OptiFine_1.12.2_HD_U_D3
	OptiFine Build: 20180515-125031
	Render Distance Chunks: 12
	Mipmaps: 0
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 398.11
	OpenGlRenderer: GeForce GTX 750 Ti/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 8

--- a/src/main/kotlin/net/shadowfacts/forgelin/Forgelin.kt
+++ b/src/main/kotlin/net/shadowfacts/forgelin/Forgelin.kt
@@ -4,7 +4,7 @@ import net.minecraftforge.fml.common.FMLCommonHandler
 import net.minecraftforge.fml.common.Loader
 import net.minecraftforge.fml.common.Mod
 import net.minecraftforge.fml.common.Mod.EventHandler
-import net.minecraftforge.fml.common.event.FMLConstructionEvent
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent
 
 /**
  * @author shadowfacts
@@ -17,9 +17,9 @@ object Forgelin {
 	const val VERSION = "@VERSION@"
 
 	@EventHandler
-	fun onConstruction(event: FMLConstructionEvent) {
+	fun onPreInit(event: FMLPreInitializationEvent) {
 		Loader.instance().modList.forEach {
-			ForgelinAutomaticEventSubscriber.subscribeAutomatic(it, event.asmHarvestedData, FMLCommonHandler.instance().side)
+			ForgelinAutomaticEventSubscriber.subscribeAutomatic(it, event.asmData, FMLCommonHandler.instance().side)
 		}
 	}
 }

